surfstudio/ios-kinescope-sdk
# Kin 19 video list

## Changes

- Add `Inpector` implementation. 
- Add unit tests with mocked `VideosApiClient`
- Add `MetaResponse` to get meta-info
- ...

## Open questions

- Tested on debug. Requesting of multiple pages will be added in next task with `ViewController` implementation.
- ...

## How to test?

- Set apiKey in `AppDelegate`
- Request list like
```
Kinescope.shared.inspector.list(request: .init(page: 1),
                                        onSuccess: { result in
                                            print("KIN result \(result)")
                                        }, onError: { error in
                                            print("KIN error \(error)")
                                        })
```
- Check logs
